lib/protocol/networks.ts

```diff
@@ -69,6 +69,7 @@ const defaultEnv = {
   hashConsensus: "HASH_CONSENSUS_ADDRESS",
   // vaults
   stakingVaultFactory: "STAKING_VAULT_FACTORY_ADDRESS",
+  stakingVaultBeacon: "STAKING_VAULT_BEACON_ADDRESS",
```

Comment:
I think it should be added to the `.env.example`. Maybe along with `*_STAKING_VAULT_FACTORY_ADDRESS` if it was not also added there.

Reply:
added